frontend/src/views/c-home.vue

```diff
@@ -120,6 +123,25 @@ const home = defineComponent({
       required: true,
     },
   },
+  data() {
+    return {
+      isTitleVisible: false,
+    };
+  },
+  computed: {
+    setScroll() {
+      return {
+        overflowY: this.isTitleVisible ? 'hidden' : 'auto',
```

Comment:
Generally, it is recommended to not have two scrolls in the same direction as "double-scrolling" causes poor UX and leads to many errors. 

The current title component integration uses double-scrolling, and I foresee many bugs in the scrolling behaviour due to this. Have mentioned one of them below:

### Steps to replicate
* Scroll to the complete bottom using the left panel
* Scroll to the top from the right panel until you can see the full title again

### Result
* The left panel does not scroll along with the right one to show the left panel's topmost contents, but the title above it is shown
* The user cannot scroll to show the top contents of the left panel unless they scroll to the bottom again until the title is not seen
* The method to scroll the full view (to show/hide the title) and the left panel is the same, so it seems like the browser randomly picks which one to scroll.

We may have to discuss with Prof @damithc regarding alternative ways to design this. I'd recommend only having the title as part of the left Summary panel, to avoid any scrolling issues.

Reply:
I agree that it's an issue and I've been unable to find a satisfactory solution to this. One thing Prof suggested was to disable panel scrolling until the title is removed from view, which was implemented, but I've been unable to resolve the double scrolling, without resorting to tricks like attempting to disable scrolling in one direction.

Would it be possible to disable title scrolling at the parts where panel scrolling is enabled, except at a certain transition point? Maybe that would cause more bugs?

Reply:
I want to clarify a few things about your suggestion:
* If title scrolling is disabled when a panel can be scrolled, at what point of time do we enable title scrolling again?
* Since both panels don't scroll together, I'm wondering how it would be possible to scroll both of them to their top when scrolling to the top again to see the title. 

I would prefer the title component to be restricted to just the left Summary panel unless this does not fit in with Prof's requirements. I think this makes scrolling easier to handle and does not affect the UI design much.

1. Across the top of the screen:

![image](https://github.com/reposense/RepoSense/assets/68203159/73617b23-c616-4399-af9f-d98e1f18dfff)

2. Only on top of the summary panel:

![301182943-e9206574-a88e-4d75-8588-bc5c3f303def](https://github.com/reposense/RepoSense/assets/68203159/244b77b2-7b97-4b50-b2f9-f66d98475305)

Reply:
I was initially thinking to enable both scrolling at the point where the title is out of view and either panel is at its top, but it would cause scrolling and UX issues. 

At this point, I agree it's probably better to just confine the panel to the left panel. I'll amend the PR to try this.